app/assets/stylesheets/libraries-global.scss

```diff
@@ -657,6 +657,9 @@ form .field-label {
   font-size: 1.4rem;
   font-weight: 600;
 }
+form .field-label.error {
+  color: #AC1D22;
```

Comment:
Flagging in case you wanted to variable this. I don't care, but I think you might? :)